leases/services/invoice.py

```diff
@@ -89,7 +89,7 @@ def create_new_lease(
             lease.contract.pk = None
             lease.contract.save()
 
-        lease.save(update_fields=["start_date", "end_date", "application", "contract"])
+        lease.save(update_fields=["start_date", "end_date", "application"])
```

Comment:
The newly created lease will still get a copy of the contract as well?

Reply:
Yup. That's created by the
```python
lease.contract.pk = None
lease.contract.save()
```

Reply:
👍 just thought to double check.